iree/base/api.h

```diff
@@ -724,17 +723,22 @@ IREE_API_EXPORT IREE_MUST_USE_RESULT iree_status_t IREE_API_CALL
 
 IREE_API_EXPORT IREE_MUST_USE_RESULT iree_status_t IREE_API_CALL
 iree_status_annotate_vf(iree_status_t base_status, const char* format,
-                        va_list varargs_0, va_list varargs_1);
+                        va_list varargs_0);
```

Comment:
you can rename this to just varargs now (_0 and _1 were disambiguators for when there were two)

Reply:
+1 (here and above)